src/lib.rs

```diff
@@ -148,6 +162,47 @@
 //! }
 //! ```
 //!
+//! ### Loading enums
+//!
+//! `axum_typed_multipart` also supports enum parsing by deriving the [`TryFromField`] trait:
+//!
+//! ```rust
+//! use axum_typed_multipart::{TryFromField, TryFromMultipart};
```

Comment:
After the new changes are rebased you will notice I started embedding the examples directly in the docs so if you want to here you could also use:
```rust
#![doc = include_str!("../examples/type_safe_enum.rs")]
```

Reply:
I don't think that it is necessary here, since example is small enough and concept is simple to demonstrate

Reply:
Yeah fair enough that makes sense